internal/sequencer/types.go

```diff
@@ -218,6 +218,8 @@ type TransferOpts struct {
 	ToAddress string
 	// Amount is the amount to be transferred
 	Amount string
+	// SequencerChainId is the chain ID of the sequencer
+	SequencerChainId string
```

Comment:
nit, but the use of initialisms in names should be uppercased https://go.dev/wiki/CodeReviewComments#initialisms